src/screens/GroupInfo/GroupMedia.js

```diff
@@ -79,12 +80,12 @@ const GroupMedia = () => {
 export default GroupMedia;
 
 const styles = StyleSheet.create({
-  container: {flex: 1, backgroundColor: '#fff'},
+  container: {flex: 1, backgroundColor: COLORS.white},
 
   toptabcontainer: {
     flexDirection: 'row',
     backgroundColor: colors.white,
-    borderBottomColor: '#00000050',
+    borderBottomColor: COLORS.black30percent,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There seems to be a typo on line 87. The color should be fetched from the `COLORS` object, not the undefined `colors`.
```diff
-     backgroundColor: colors.white,
+     backgroundColor: COLORS.white,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->